### PR TITLE
[WFLY-16788] Move org.wildfly.security:wildfly-elytron-jaspi from tes…

### PR DESCRIPTION
**Pull request by yersan**
…tsuite/integration to the legacy modules

Jira issue: https://issues.redhat.com/browse/WFLY-16788

### CODE REVIEW

File: testsuite/integration/legacy-ejb-client/pom.xml [hunk 1]
```diff
@@ -64,6 +64,11 @@
             <artifactId>jakarta.interceptor-api</artifactId>
             <scope>test</scope>
         </dependency>
+        <dependency>
+            <groupId>org.wildfly.security</groupId>
+            <artifactId>wildfly-elytron-jaspi</artifactId>
+            <scope>test</scope>
+        </dependency>
```
Comment (bstansberry) [inline testsuite/integration/legacy-ejb-client/pom.xml:71]: Please remove. My testing verifies this dep is not needed.

File: testsuite/integration/legacy/pom.xml [hunk 2]
```diff
@@ -53,6 +53,11 @@
             <artifactId>jboss-jms-api_2.0_spec</artifactId>
             <scope>test</scope>
         </dependency>
+        <dependency>
+            <groupId>org.wildfly.security</groupId>
+            <artifactId>wildfly-elytron-jaspi</artifactId>
+            <scope>test</scope>
+        </dependency>
```
Comment (bstansberry) [inline testsuite/integration/legacy/pom.xml:60]: Please remove. My testing verifies this dep is not needed.

**Review by bstansberry — CHANGES_REQUESTED** [review]
@yersan Looks like all that's needed is removal of the dep from testsuite/integration/pom.xml; the child modules don't need it.

Except... I think it can be removed from boms/legacy-ee as well, since the only use of it is in testsuite/integration/pom.xml.

**Comment by yersan** [comment]
Thanks for the review @bstansberry , I've removed both dependencies

**Review by bstansberry — APPROVED** [review]

**Comment by bstansberry** [comment]
Thanks @yersan

**Pull request merged**